tests/Unit/CBARTest.php

```diff
@@ -125,17 +127,11 @@ public function test_method_get_method_get_rates_from_cbar_if_rates_for_given_da
 
     public function test_magic_set_method_returns_calculated_amount()
     {
+        // why get method was not used to retrieve the data?
+        // Here is an example.
         $cbar = new CBAR('01.05.2019');
-        $cbar->setRates([
-            '01.05.2019' => [
-                'USD' => [
-                    'nominal' => 1,
-                    'rate'    => 1.7053,
-                ],
-            ],
-        ]);
-
-        $this->assertEquals(170.53, $cbar->USD(100));
+        $cbar->getRates();
+        $this->assertEquals(170.0, $cbar->USD(100));
```

Comment:
Please revert back changes on this test case. `getRates()` has no effect here. `$cbar->USD(100)` returns actual, real rates that we have no control over.